ReClass.NET/Plugins/PluginManager.cs

```diff
@@ -63,10 +65,10 @@ private void LoadPlugins(IEnumerable<FileInfo> files, ILogger logger)
 				{
 					fvi = FileVersionInfo.GetVersionInfo(fi.FullName);
 
-					if (fvi.ProductName != PluginInfo.PluginName && fvi.ProductName != PluginInfo.PluginNativeName)
+					/*if (fvi.ProductName != PluginInfo.PluginName && fvi.ProductName != PluginInfo.PluginNativeName)
```

Comment:
Maybe a parameter `bool checkProductName` should be used for `LoadPlugins` to keep the check for supported files (exe, dll).

Reply:
Yea, the version info isnt present at all under linux which made it skip my native plugins at all. not sure what the best solution would be here

Reply:
we could add the parameter and add a mono specific check to LoadPlugins depending on the OS